config/sync/field.field.taxonomy_term.layer_type.layer_type_id.yml

```diff
@@ -0,0 +1,19 @@
+uuid: d0f0279c-469e-4fa2-bb5f-02d75bff9d25
+langcode: en
+status: true
+dependencies:
+  config:
+    - field.storage.taxonomy_term.layer_type_id
+    - taxonomy.vocabulary.layer_type
+id: taxonomy_term.layer_type.layer_type_id
+field_name: layer_type_id
```

Comment:
This field is for storing the machine name of the computed field, so layer_type_id is not the right name for it. It should be something like 'computed_geolayer_field'.
Please add a field description as well that describes what this field is used for.